tests/filters/test_gaussian.py

```diff
@@ -14,6 +15,35 @@
 from testing.base import BaseTester, assert_close
 
 
+@pytest.mark.parametrize(
+    "window_size, sigma, mean, expected",
+    [
+        (5, 1.0, None, torch.tensor([[0.0545, 0.2442, 0.4026, 0.2442, 0.0545]])),
+        (
+            11,
+            5.0,
+            None,
+            torch.tensor([[0.0663, 0.0794, 0.0914, 0.1010, 0.1072, 0.1094, 0.1072, 0.1010, 0.0914, 0.0794, 0.0663]]),
+        ),
+        (
+            11,
+            5.0,
+            8.0,
+            torch.tensor([[0.0343, 0.0463, 0.0600, 0.0747, 0.0895, 0.1029, 0.1138, 0.1208, 0.1232, 0.1208, 0.1138]]),
+        ),
+        (
+            11,
+            11.0,
+            3.0,
+            torch.tensor([[0.0926, 0.0946, 0.0957, 0.0961, 0.0957, 0.0946, 0.0926, 0.0900, 0.0867, 0.0828, 0.0785]]),
+        ),
+    ],
+)
+def test_gaussian(window_size, sigma, mean, expected):
+    result = gaussian(window_size, sigma, mean=mean)
+    assert_close(result.sum(), expected.sum())
```

Comment:
To avoid creating tensors on tests suite build (or discovery?), and also be able to test with device and dtype...

why are you asserting the sums? I think it may fail because of the expected precision, if this is the case we can use `atol`/`rtol` to change the assert_close tolerance